frontend/integration-tests/protractor.conf.ts

```diff
@@ -15,6 +15,7 @@ import {
 
 const tap = !!process.env.TAP;
 
+export const BROWSER_NAME = process.env.E2E_BROWSER ? process.env.E2E_BROWSER : 'chrome';
```

Comment:
We should prefix this with `BRIDGE_` like other environment variables. (In hindsight, we should have done that for `JASMINE_TIMEOUT` as well.

We might want to put `PROTRACTOR` in the name unless we think we can support this for Cypress as well. cc @dtaylor113 

```suggestion
export const BROWSER_NAME = process.env.BRIDGE_E2E_BROWSER_NAME || 'chrome';
```

Reply:
Ah, clearly a better syntax to do the same. And we want to pick the appropriate name. Let me know and I'll update the commit.

Reply:
@miyamotoh I'd suggest `BRIDGE_PROTRACTOR_BROWSER_NAME` to make it really clear.

Reply:
Roger that, a force-push coming with the new env var and formatting changes..

Reply:
Oh, do you want `JASMINE_TIMEOUT` to be renamed to `BRIDGE_JASMINE_TIMEOUT` in this PR? I can easily do that. Let me know.

Reply:
Yeah, I think that's a good change.